src/main/java/org/sagebionetworks/bridge/sdk/models/users/UserProfile.java

```diff
@@ -8,19 +8,22 @@
     private String firstName;
     private String lastName;
     private String username;
+    private String phone;
     private String email;
 
     @JsonCreator
     private UserProfile(@JsonProperty("firstName") String firstName, @JsonProperty("lastName") String lastName,
-            @JsonProperty("username") String username, @JsonProperty("email") String email) {
+            @JsonProperty("username") String username, @JsonProperty("email") String email,
+            @JsonProperty("phone") String phone) {
         this.firstName = firstName;
         this.lastName = lastName;
         this.username = username;
         this.email = email;
+        this.phone = phone;
     }
     
     public static UserProfile valueOf() {
```

Comment:
valueOf()... since these classes are mutable, what's the purpose of this? Maybe we should remove these from all model objects in SDK.